clients/privacy-center/packages/fides-consent/src/fides-consent.ts

```diff
@@ -11,22 +11,24 @@ import { meta } from "./integrations/meta";
 import { shopify } from "./integrations/shopify";
 import { ConsentConfig } from "./lib/consent-config";
 import { getConsentContext } from "./lib/consent-context";
-import { getConsentCookie, makeDefaults } from "./lib/cookie";
+import { getOrCreateFidesCookie, makeConsentDefaults } from "./lib/cookie";
 
 const config: ConsentConfig = consentConfig;
 const context = getConsentContext();
-const defaults = makeDefaults({
+const defaults = makeConsentDefaults({
   config,
   context,
 });
 
 /**
  * Immediately load the stored consent settings from the browser cookie.
  */
-const consent = getConsentCookie(defaults);
+const cookie = getOrCreateFidesCookie(defaults);
 
 const Fides = {
-  consent,
+  consent: cookie.consent,
+  fides_meta: cookie.fides_meta,
+  identity: cookie.identity,
```

Comment:
See cookie.ts for the updated format of this cookie; I effectively just destructure it here and add to the `Fides` global for convenience.